pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -241,7 +244,7 @@ def post_dispatch(self) -> None:
         # todo, pass also bets score
 
         # load last weights
-        if last_path and not self.lightning_module.trainer.testing:
+        if last_path and model.trainer.state != TrainerState.FITTING:
```

Comment:
can't wrap my head around this. 
Before it was loading when not testing, and now it is loading when not fitting.

Reply:
Yep, looks wrong